frontend/src/scenes/web-analytics/tiles/WebAnalyticsTile.tsx

```diff
@@ -463,7 +464,20 @@ export const WebStatsTableTile = ({
                         <LemonSwitch
                             label={
                                 <div className="flex flex-row space-x-2">
-                                    <span>Enable path cleaning</span>
+                                    <Tooltip
```

Comment:
IMO we don't need this tooltip AND a link to the path cleaning setting, though maybe the tooltip should move to where the settings live

Reply:
I thought it was quite confusing what this did, but it's true that if you click on the settings button you might learn about it. I'll try to find an alternative solution to make this easy to read, but I admit I'm tending towards just keeping this tooltip here, I don't think it hurts.

Reply:
I kept the tooltip here, but added the link to the docs in the actual path cleaning configuration page. I've added the link even in case people don't have access to the feature yet, this way it might push people to adopt new advanced features.